spec/system/accessibility_spec.rb

```diff
@@ -22,7 +22,6 @@
   context 'when viewing claim-specific screens' do
     %i[nsm_claim_claim_details
        nsm_claim_adjustments
-       nsm_claim_letters_and_calls
```

Comment:
Given that you've fixed what this spec was complaining about, I don't think you need to remove this here any more, do you? It seems to pass on my machine when I put this line back in.

Reply:
it's a duplicate - it was failing in two spots... the grouping is a little off, but descided it was better the way I left it.